deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2

```diff
@@ -29,6 +41,7 @@ services:
       - ./logs:/var/log/
       - ./tmp:/tmp
       - {{ integration_test.inasafe_headless.test_data_path }}:{{ integration_test.inasafe_headless.mapped_test_data_path }}
+      - {{ inasafe_realtime.host_working_dir }}:{{ inasafe_realtime.working_dir }}
```

Comment:
@ismailsunni perhaps we're thinking the same thing. But I'm not sure if your config for headless up there were used only for integration test? For mine, it is needed to setup local dev too. Since it is not only for integration_test, I didn't use that key and directly used `inasafe_realtime` key instead. We have to agree on this.

Reply:
Yes, I think we are thinking the same. I will follow your convention. I named it integration_test since I need it to run the integration test while developing locally. So, it's the same.